src/App/App.js

```diff
@@ -33,8 +33,10 @@ class App extends Component {
 
   setUser(user){
     this.setState(user);
-    getAllRatings(this.state.user.id)
-    .then(ratings => this.setState({userRatings: ratings.ratings}))
+    if (this.state.user.name) {
```

Comment:
I appreciate that this condition is added here, since we don't want to make the fetch request if we're certain the user has missing information.